var/spack/repos/builtin/packages/vtk/package.py

```diff
@@ -120,9 +120,9 @@ class Vtk(CMakePackage):
     depends_on("gl@1.2:", when="~opengl2")
 
     depends_on("xz")
-    patch("vtk_find_liblzma.patch", when="@:9.0")
-    patch("vtk_movie_link_ogg.patch")
-    patch("vtk_use_sqlite_name_vtk_expects.patch")
+    patch("vtk_find_liblzma.patch", when="@8")
```

Comment:
```console
> spack patch vtk@8.0.1
==> Fetching https://mirror.spack.io/_source-cache/archive/49/49107352923dea6de05a7b4c3906aaf98ef39c91ad81c383136e768dcf304069.tar.gz
==> Applied patch /Users/Adam/spack/var/spack/repos/builtin/packages/vtk/internal_findHDF5.patch
File to patch: 
No file found--skip this patch? [y] 
1 out of 1 hunks ignored--saving rejects to Oops.rej
==> Patch /Users/Adam/spack/var/spack/repos/builtin/packages/vtk/vtk_find_liblzma.patch failed.
==> Error: Command exited with status 1:
'/usr/bin/patch' '-s' '-p' '1' '-i' '/Users/Adam/spack/var/spack/repos/builtin/packages/vtk/vtk_find_liblzma.patch' '-d' '.'
```

Reply:
Ping @johnwparent

Reply:
Well TIL about the `spack patch` command.

Trying to scope to `@8.2` since older versions (and newer versions) don't work on Windows ATM anyway.

Reply:
`spack patch` is very convenient, you can even do something like:
```bash
for v in $(spack versions -s vtk)
do 
    spack patch vtk@$v
done
```
to quickly test all versions.

I'm fine with limiting to 8.2 if that's what you want.

Reply:
Neat! Thanks, this is definitely making it into the package "out of the box Windows support" testing workflow I have setup. 

I can't test with earlier than 8.2 as `netcdf-cxx4` hasn't had its CMake system integrated into Spack yet (I'm currently 3 patches deep into their CMake system to get Windows supported) (and they clearly don't apply for you) and these don't apply to `vtk@9` so I think this is the correct version range anyway.

Reply:
@adamjstewart thanks again, already used that script twice today.